avocado/core/status/server.py

```diff
@@ -1,5 +1,7 @@
 import asyncio
 
+from ..settings import settings
```

Comment:
do you mind to use absolute imports here?

Reply:
But we're not using absolute imports on `avocado/core`...

Reply:
main.py and settings.py are using so I thought that was already ok, but Ok, no worries here.